tests/meta/demultiplex/test_housekeeper_storage_functions.py

```diff
@@ -213,3 +216,30 @@ def test_add_demux_logs_to_housekeeper(
     assert len(files) == 2
     for file in files:
         assert file.path.split("/")[-1] in expected_file_names
+
+
+def test_store_fastq_path_in_housekeeper_correct_tags(
+    populated_housekeeper_api, empty_fastq_file_path, novaseq6000_flow_cell, store
+):
+    """Test that a fastq file is stored in Housekeeper with the correct tags."""
+    sample_id: str = "sample_internal_id"
+    # GIVEN a fastq file that has not been added to Housekeeper
+    assert not populated_housekeeper_api.files(path=empty_fastq_file_path.as_posix()).first()
+
+    # WHEN adding the fastq file to housekeeper
+    with mock.patch(
+        "cg.meta.demultiplex.housekeeper_storage_functions.check_if_fastq_path_should_be_stored_in_housekeeper",
+        return_value=True,
```

Comment:
Mocking internal function calls like this is not great, if it is possible to write the test without this mock it is better.

Reply:
It is bad because:
1. The test will break if something about the mocked function changes (name, moved to a difference module).
2. The test is tightly coupled to the implementation of the function instead of the behaviour of it. The implementation is likely to change, the behaviour is not.

Reply:
It makes sense. What do you think about the fix @seallard ?

Reply:
I think it looks great 👍  If you are happy with it as well, lets go for it

Reply:
keeping the fix